Header stdMsgHeader
int32 chargingStationID
---
Header stdMsgHeader
msg1 messageContent
